configs/config.go

```diff
@@ -14,6 +14,9 @@ This is an initial implementation of Config. Please keep in mind we're still wor
 environment variables and general config data.
 */
 
+// to use with parans ldlfags
+var k1Version string
+
 // Config host application configuration
 // todo: some of these values can be moved to the .env
```

Comment:
**How we run for development?**

Do we just export this on our terminal before runs:
```
export LDFLAGS="-X github.com/kubefirst/kubefirst/configs.k1Version=1.8.6"
```